Batch/src/V1/Gapic/BatchServiceGapicClient.php

```diff
@@ -1018,200 +1011,4 @@ public function listLocations(array $optionalArgs = [])
             'google.cloud.location.Locations'
         );
     }
-
-    /**
-     * Gets the access control policy for a resource. Returns an empty policy
-    if the resource exists and does not have a policy set.
-     *
-     * Sample code:
-     * ```
-     * $batchServiceClient = new BatchServiceClient();
-     * try {
-     *     $resource = 'resource';
-     *     $response = $batchServiceClient->getIamPolicy($resource);
-     * } finally {
-     *     $batchServiceClient->close();
-     * }
-     * ```
-     *
-     * @param string $resource     REQUIRED: The resource for which the policy is being requested.
-     *                             See the operation documentation for the appropriate value for this field.
-     * @param array  $optionalArgs {
-     *     Optional.
-     *
-     *     @type GetPolicyOptions $options
-     *           OPTIONAL: A `GetPolicyOptions` object for specifying options to
-     *           `GetIamPolicy`.
-     *     @type RetrySettings|array $retrySettings
-     *           Retry settings to use for this call. Can be a {@see RetrySettings} object, or an
-     *           associative array of retry settings parameters. See the documentation on
-     *           {@see RetrySettings} for example usage.
-     * }
-     *
-     * @return \Google\Cloud\Iam\V1\Policy
-     *
-     * @throws ApiException if the remote call fails
-     */
-    public function getIamPolicy($resource, array $optionalArgs = [])
```

Comment:
Removing these methods is a breaking change, however this API is pre-GA, so this is allowed. We do need to confirm that these changes are expected, however.